spec/factories/user_profiles.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :user_profile do
```

Comment:
I think Its better if the user creates that in the test itself(controller spec file) as the test can vary for different kinds of user.